newIDE/app/src/BehaviorsEditor/index.js

```diff
@@ -343,15 +573,36 @@ const BehaviorsEditor = (props: Props) => {
             })}
           </ScrollView>
           <Column>
-            <Line justifyContent="flex-end" expand>
-              <RaisedButton
-                key="add-behavior-line"
-                label={<Trans>Add a behavior</Trans>}
-                primary
-                onClick={() => setNewBehaviorDialogOpen(true)}
-                icon={<Add />}
-                id="add-behavior-button"
-              />
+            <Line noMargin>
```

Comment:
I think you'll need another responsive line stack layout here

Reply:
![image](https://user-images.githubusercontent.com/2611977/230632478-6d2b485e-1123-4822-9952-c70cb7315862.png)

I decided to remove the text because 3 row of button was looking weird.